website/content/docs/configuration/server_join.mdx

```diff
@@ -113,24 +129,43 @@ unless configured otherwise:
 nomad-01.company.local => nomad-01.company.local:4648
 ```
 
-### Via the go-discover interface
+### `go-discover` format
 
 As of Nomad 0.8.4, `retry_join` accepts a unified interface using the
 [go-discover](https://github.com/hashicorp/go-discover) library for doing
-automated cluster joining using cloud metadata. See [Cloud
+automated cluster joining using cloud provider metadata. See [Cloud
 Auto-join](#cloud-auto-join) for more information.
 
 ```
 "provider=aws tag_key=..." => 1.2.3.4:4648
 ```
 
+### `go-netaddrs` format
+
+As of Nomad 1.x.x (**TODO**: What version would this land in?) `retry_join` accepts
```

Comment:
Calling out this TODO for a response.

Reply:
We have versioned docs these days, so this would only appear in the major version in which the feature appears (ex. the 1.7.x docs). If we can manage to land this PR in 1.7.0, which seems likely, I think we can remove this clause entirely. Let's leave it out and if we need to adjust it I can come back in later to tweak it.

Reply:
Excellent. That sounds much better.